tests/unit/test_cli.py

```diff
@@ -228,13 +230,54 @@ def __init__(self, outputFormat='text'):
             self.baseUrl = 'baseUrl'
             self.verbose = 'verbose'
 
-    class FakeObject(protocol.ProtocolElement):
-
-        __slots__ = ['id', 'name']
-
-        def __init__(self):
-            self.id = 'id'
-            self.name = 'name'
+    class FakeObject(protocol.message.Message):
```

Comment:
This looks pretty brittle to me. I think we can just delete the ``TestOutputFormats`` test case. It's not particularly strong, and it's more trouble that it's worth if we have to go rummaging in the guts of protobuf.